tests/test_integration.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import unittest
+import os
+import sys
+
+from dwave.cloud.utils import retried
+
+project_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# These integration tests are used to validate example run successfully with other integrated systems
+# It is designed to run from any other directory
+
+class IntegrationTests(unittest.TestCase):
```

Comment:
Why remove the skip decorator?
```
@unittest.skipUnless(hybrid_solver_available(), ...
```

Reply:
@randomir These are integration tests and need to run it from location other than project directory

When I run your integration tests from test_equivalence.py
I am getting error
```
ImportError while importing test module '/home/hpawar/sapi/test-automation/tests/components/ide/test_ide_examples_suite_bqm.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/components/ide/test_ide_examples_suite_bqm.py:4: in <module>
    from circuit_equivalence.tests.test_equivalence import IntegrationTest as circuit_equivalence
circuit_equivalence/tests/test_equivalence.py:22: in <module>
    from tests import hybrid_solver_available
E   ImportError: cannot import name 'hybrid_solver_available'
```

Reply:
I need to create a suite of all the integration tests from different examples and run it together to get better html report
I do that by creating a python file and importing all integrations tests
```
from n_queens1.tests.test_n_queens import TestSmoke as n_queens
from satellite_placement.tests.test_satellite import TestSmoke as satellite_placement
from knapsack.tests.test_knapsack import IntegrationTest as knapsack
from circuit_equivalence.tests.test_equivalence import IntegrationTest as circuit_equivalence
```
and then call as python -m py.test tests/components/ide/test_ide_examples_suite
in my jenkins file

If I run integration tests with along with other tests , it throws error

Reply:
It will certainly help me to have integration in separate file but the tests I run are internal to dwave and
examples are public so I will leave upto you guys how to organize these tests.

Reply:
This approach of isolating integration test(s) and removing all relative imports simply does not scale (and the first hint is the need for `tests.hybrid_solver_available`).

I'd suggest you simply run:
```
python -m unittest discover
```
from a parent directory that contains all examples.

That will run _all tests_ (including integration tests) on _all examples_, but getting a more comprehensive coverage shouldn't be a problem, right?